<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshElementRemoval</class>
 <widget class="QDialog" name="MeshElementRemoval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Mesh Elements</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0">
   <item row="7" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="meshNameComboBox"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="meshNameLabel">
     <property name="text">
      <string>Select mesh to edit:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Remove Elements based on</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="yOutsideLabel">
        <property name="text">
         <string>Y outside of</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="yAndLabel">
        <property name="text">
         <string>and</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="xMaxEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" alignment="Qt::AlignTop">
       <widget class="QCheckBox" name="materialIDCheckBox">
        <property name="text">
         <string>Material ID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="xOutsideLabel">
        <property name="text">
         <string>X outside of</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" alignment="Qt::AlignTop">
       <widget class="QCheckBox" name="elementTypeCheckBox">
        <property name="text">
         <string>Element Type</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="zeroVolumeCheckBox">
        <property name="text">
         <string>Zero Area/Volume</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="zOutsideLabel">
        <property name="text">
         <string>Z outside of</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="xAndLabel">
        <property name="text">
         <string>and</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="zMinEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="yMaxEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="zAndLabel">
        <property name="text">
         <string>and</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="xMinEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="yMinEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLineEdit" name="zMaxEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="boundingBoxCheckBox">
        <property name="text">
         <string>Bounding Box, Nodes with</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QListWidget" name="elementTypeListWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <item>
         <property name="text">
          <string>Line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tri</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quad</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tetrahedron</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hexahedron</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pyramid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prism</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QListWidget" name="materialListWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="newMeshNameLabel">
     <property name="text">
      <string>Name of new mesh:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="newMeshNameEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeshElementRemoval</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeshElementRemoval</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
